test/gen/scales.test.ts

```diff
@@ -5,14 +5,18 @@ import {expect} from 'chai';
 describe('cp.gen.scales()', function () {
   it('should correctly generate fieldSets with scale variation', function() {
```

Comment:
Ideally, I would write another test case to test the bin case like:

`it(’should not generate scale variation for bin’)`

and just pass one field to the function